Gaijinizer patch/Map469.txt

```diff
@@ -3,721 +3,912 @@
+> CONTEXT: Map469/events/3/pages/0/336/Dialogue/enemy38
+Kuh... I suppose I have no choice...
 
+It would seem you've narrowly escaped death,
+\\N[1]. But our battle is yet to come!!#RonLocalization
 > END STRING
 
 > BEGIN STRING
 そしてマリオン…！！！
 
 アンタとはやっぱり決着を付ける必要があるみたいね…！！
 今に見てなさい！！
-> CONTEXT: Map469/events/3/pages/0/341/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/341/Dialogue/enemy38
+And as for you, Marion!!!
 
+It seems we have a score to settle!! Just you
+watch!!#Ron
 > END STRING
 
 > BEGIN STRING
 よっ、と。
-> CONTEXT: Map469/events/3/pages/0/387/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/387/Dialogue/sirousagi_fc1
+There we go...#Ron
 > END STRING
 
 > BEGIN STRING
 マリオンくんは喋らないよ。
 
 別に喋れないってワケじゃないけど
 ほら、『無言系』ってやつ？
-> CONTEXT: Map469/events/3/pages/0/389/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/389/Dialogue/sirousagi_fc1
+Marion-kun doesn't talk.
 
+It's not that he can't talk, but he's what you'd
+call, you know, the "silent type".#Ron
 > END STRING
 
 > BEGIN STRING
 無言系……
-> CONTEXT: Map469/events/3/pages/0/394/Dialogue/ruka_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/394/Dialogue/ruka_fc1
+The silent type...#Ron
 > END STRING
 
 > BEGIN STRING
 無言系か…
-> CONTEXT: Map469/events/3/pages/0/396/Dialogue/alice_fc5 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/396/Dialogue/alice_fc5
+The silent type, huh...#Ron
 > END STRING
 
 > BEGIN STRING
 無言系かぁ
-> CONTEXT: Map469/events/3/pages/0/398/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/398/Dialogue/enemy15
+So he's the silent type...#Ron
 > END STRING
 
 > BEGIN STRING
 …………無言系？
-> CONTEXT: Map469/events/3/pages/0/400/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/400/Dialogue/kazuya
+......The silent type?#Ron
 > END STRING
 
 > BEGIN STRING
 さっき『最後のゲスト』って言ってたけど……
 
 それじゃあ、彼がこの世界で見つける予定だった子？
 随分と年齢は若いように見えるけど……
-> CONTEXT: Map469/events/3/pages/0/402/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/402/Dialogue/enemy38
+You said just before that he's the last guest...
 
+Does that mean he's the one we were planning on
+finding in this world? He looks quite young.#Ron
 > END STRING
 
 > BEGIN STRING
 中学生でその身なりのミズキちゃんに言われると
 返す言葉に困るなぁ……
 
 彼はキミより年上だよ。
-> CONTEXT: Map469/events/3/pages/0/407/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/407/Dialogue/sirousagi_fc1
+I don't know how to respond when a middle-schooler
+as young as you says something like that...
 
+He's actually older than you.#Ron
 > END STRING
 
 > BEGIN STRING
 天使の寵愛を受けた、聖騎士『サイトウ』……
 
 その子孫であるマリオンくんは、かつて天使のオーブを巡る
 天使と魔物の騒乱を解決した人間なんだ。
-> CONTEXT: Map469/events/3/pages/0/412/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/412/Dialogue/sirousagi_fc1
+Marion is the descendant of the holy knight
+"Saitou" who had the blessings of the angels.
 
```

Comment:
"Saitou" or Saito? Both have been used before this.

Reply:
Should be Saito, I'll fix

Reply:
Cool, I'll trust your judgement on this.